src/distrs/beta.jl

```diff
@@ -1,7 +1,6 @@
 # functions related to beta distributions
 
 import .RFunctions:
-    betalogpdf,
```

Comment:
You don't want to remove this one and you probably want to add `betapdf`. Same with `gamma`. The new definitions should only be for non `Float64` elements.